compiler/src/main/scala/dx/compiler/ExecutableCompiler.scala

```diff
@@ -51,7 +52,9 @@ object ExecutableCompiler {
 
 class ExecutableCompiler(extras: Option[Extras],
                          parameterLinkSerializer: ParameterLinkSerializer,
-                         dxApi: DxApi = DxApi.get) {
+                         dxApi: DxApi = DxApi.get,
```

Comment:
Why are project and folder included here if they are not used anywhere in ExecutableCompiler?